regtests/bin/matrix.base

```diff
@@ -716,20 +716,9 @@
       echo "$rtst -s PR3_UQ              -w work_PR3_UQ                 $ww3 ww3_tp2.13" >> matrix.body
       echo "$rtst -o netcdf              -w work_PR3_UQ                 $ww3 ww3_tp2.15" >> matrix.body
       echo "$rtst -o netcdf              -w work_5km -g 5km             $ww3 ww3_tp2.15" >> matrix.body
-      echo "$rtst -o netcdf -i input_rho -w work_PR3_UQ_RHO -r ww3_grid $ww3 ww3_tp2.15" >> matrix.body
```

Comment:
There are a lot of changes here. I assume this is the change we made to avoid running all the ww3_tp2.15 programs separately? Maybe mention that in the PR description?

Reply:
the PR description now includes:
"Other changes included here:
Includes the correction of the typo in OASACM switch (L2264 in ww3_shel.ftn).
Changes in matrix.base to avoid running all the ww3_tp2.15 programs separately"